src/RRule.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+namespace ipl\Scheduler;
+
+use BadMethodCallException;
+use DateTime;
+use DateTimeInterface;
+use Generator;
+use ipl\Scheduler\Contract\Frequency;
+use Recurr\Rule as RecurrRule;
+use Recurr\Transformer\ArrayTransformer;
+use Recurr\Transformer\ArrayTransformerConfig;
+use Recurr\Transformer\Constraint\AfterConstraint;
+
+use function ipl\Stdlib\get_php_type;
+
+class RRule implements Frequency
+{
+    /** @var string Run once a year */
+    public const YEARLY = 'YEARLY';
```

Comment:
What are we doing with all the constants?

Reply:
Are being used by several config forms.